UPGRADE_NOTES.md

```diff
@@ -144,11 +144,11 @@ end
 - Rename: `fillable` to `permit_columns`
 - Rename: form class names to new naming convention. (e.g. `class UserForm < User::SaveOperation` => `class SaveUser < User::SaveOperation`). This step is optional, but still recommended to avoid future confusion.
 - Rename: `Avram::VirtualForm` to `Avram::Operation`.
-- Rename: virtual form class names to new naming convention. (e.g. `class SignInForm < Avram::Operation` => `class SignInUser < Avram::Operation`).
+- Rename: virtual form class names to new naming convention NounVerb. (e.g. `class SignInForm < Avram::Operation` => `class SignInUser < Avram::Operation`).
```

Comment:
This should be `VerbNoun` (same on line 151)

Reply:
oops! my bad.